### DISCUSSION TITLE
NAppGUI is using UTF-32 for Unicode?

### DISCUSSION THREAD

**Discussion by ghost** [frang75/nappgui_src#62] [General]
Please correct me if I'm wrong. But if that's the case, do you think it's a waste of memory? The world has embraced UTF-8, and there are plenty of libraries to work with it already.

**Comment by frang75** [comment]
Hi @iahung2. Thanks for commenting.

NAppGUI uses UTF-8 in all its methods. It says this clearly in several points of its documentation. Another different thing is that it provides conversion methods to/from UTF16 and UTF32.

https://nappgui.com/en/sewer/unicode.html#h1.4

![image](https://github.com/frang75/nappgui_src/assets/42999199/7a6f49fc-7f3c-4840-a552-3ea3decbbe87)

![image](https://github.com/frang75/nappgui_src/assets/42999199/21ba16ec-4b2d-4aea-8962-bcb20bac0e7a)
